addons/account/models/chart_template.py

```diff
@@ -933,6 +933,24 @@ def _get_tax_vals(self, company, tax_template_to_tax):
             val['tax_group_id'] = self.tax_group_id.id
         return val
 
+    def get_tax_vals_complete(self, company):
+        """
+        Returns a dict of values to be used to create the tax corresponding to the template, assuming the
+        account.account objects were already created.
+        The existing method _get_tax_vals is used when the account.account are not yet created. Thus, we need to add the
+        missing field cash_basis_transition_account_id. The invoice and refund repartition lines are also updated
+        such that they contain the following account.account objects: account_id, invoice_tax_id and refund_tax_id.
+        (Used by upgrade/migrations/util/accounting)
+        """
+        vals = self._get_tax_vals(company, {})
+        vals.pop("children_tax_ids", None)
+        vals.update({
+            "cash_basis_transition_account_id": self.cash_basis_transition_account_id.id,
```

Comment:
But this is an account.account.template, not an account.account, right ?

Reply:
indeed, I now browse the record to retrieve it